README.md

```diff
@@ -132,6 +132,11 @@ Technology - Lately I prefer to use React with Redux and webpack, but I'm also f
 #### JS Questions:
 
 * Explain event delegation
+    Event delegation consists in bundling an event handler into a parent rather than an element that we want to handle.
+    When we click on an interior element, the event bubbles up to the item assigned to the handler.
+    Then, with event.target, we can see where the event occurred and execute it on the corresponding element.
+    This allows us to handle events occurring on multiple items with one handler.
```

Comment:
Jest to odpowiedź z pliku, który dałaś na kanał. Zweryfikowałem to z tekstem na tej stronie i wg. mnie się pokrywa:
http://www.crimsteam.site90.net/crimsteam/dom/dom_zdarzenia_delegacja.html